redbot/cogs/streams/streams.py

```diff
@@ -109,22 +109,28 @@ async def move_api_keys(self) -> None:
     async def get_twitch_bearer_token(self) -> None:
         tokens = await self.bot.get_shared_api_tokens("twitch")
         if tokens.get("client_id"):
+            notify_owner = await self.config.notified_owner()
             try:
                 tokens["client_secret"]
+                if notify_owner is True:
+                    await self.config.notified_owner.set(False)
             except KeyError:
                 message = _(
-                    "You need a client secret key to use correctly Twitch API on this cog.\n"
+                    "You need a client secret key to correctly use the Twitch API on this cog.\n"
```

Comment:
"to correctly use" seems weird here, maybe we could just use "if you want to" or "to be able to"?
```suggestion
                    "You need a client secret key if you want to use the Twitch API on this cog.\n"
```